xbmc/utils/ProgressJob.h

```diff
@@ -158,6 +158,6 @@ class CProgressJob : public CJob
   bool m_autoClose = true;
   bool m_updateProgress = true;
   bool m_updateInformation = true;
-  CGUIDialogProgressBarHandle* m_progress;
-  CGUIDialogProgress* m_progressDialog;
+  CGUIDialogProgressBarHandle* m_progress = nullptr;
+  CGUIDialogProgress* m_progressDialog = nullptr;
```

Comment:
Alright, and now we can remove init from the different ctors, right?

https://github.com/xbmc/xbmc/blob/master/xbmc/utils/ProgressJob.cpp#L21-L22
https://github.com/xbmc/xbmc/blob/master/xbmc/utils/ProgressJob.cpp#L27

Icing on the cake would be to remove this as well:
https://github.com/xbmc/xbmc/blob/master/xbmc/utils/ProgressJob.cpp#L27